src/Card/Card.jsx

```diff
@@ -6,6 +6,7 @@ import CardHeader from './CardHeader';
 import buildThemeProperties from '../Utils';
 
 import style from '../Layout/LayoutStyle.scss';
+import PlaceholderTextComponent from '../../dev/src/components/PlaceholderComponent';
```

Comment:
why is this in the '/dev/src' directory? Production code should not be in '/dev', it should be under '/src/'

Reply:
Ok.